docs/diagrams/EditProcCommand.puml

```diff
@@ -0,0 +1,105 @@
+-> ":UiManager" : start()
```

Comment:
The color of the diagram should be consistent with other ones. Also, the storage part may not be that necessary. What is your thought on that? Other than these, it looks fine!

Reply:
Thank you! This is a great comment, I have since changed up my formatting :).